examples/next-openai-pages/pages/index.tsx

```diff
@@ -36,6 +36,14 @@ const examples = [
     title: 'Stream chat completion (edge runtime)',
     link: '/chat/stream-chat-edge',
   },
+  {
+    title: 'Stream chat completion with attachments',
+    link: '/chat/stream-chat-attachments',
+  },
+  {
+    title: 'Stream chat completion with attachments (url)',
+    link: '/chat/stream-chat-attachments-url',
+  },
```

Comment:
- `use-chat-attachments`?
- move to `examples/next-openai` and leave only pages-router specific examples in pages route file?

Reply:
done!